### DISCUSSION TITLE
Lvl 40

### DISCUSSION THREAD

**Discussion by metalskatan** [isid0re/SoloLeveling#244] [Q&A] [answered]
Is there a way to get it to just get to lvl 40 in norm without killing Baal? Trying to make bumpers. If there is a way, any help would be great! Or if there's another way to do this entirely?

Thanks!

**Comment by isid0re** [comment, marked as answer]
Edit the sololeveling/scripts/baal.js file to exit before killing baal or after last wave.

**Reply by metalskatan** [reply]
Duh... I knew that! Thanks for the speedy response!

**Reply by metalskatan** [reply]
I'm sorry. I thought there would be an area to comment it out or something. How exactly would I make it stop before killing Baal without an error that is.

Thanks for your time it's really appreciated!

**Reply by isid0re** [reply]
Just exit with a return true;

**Reply by mushmasterflex** [reply]
im trying to do this as well. can u be more specific.

**Reply by isid0re** [reply]
Added bumper options

Now if you want a bumper for normal with any class. Just put `Nbumper` and for nightmare `Nmbumper` in the infotag

**Reply by isid0re** [reply]
bumper option has been changed to just Bumper
